package.json

```diff
@@ -67,12 +67,12 @@
     "ioredis": "^5.2.4",
     "jquery": "^3.5.1",
     "lodash": "^4.17.21",
-    "minimist": "^1.2.7",
+    "minimist": "^1.2.6",
     "moment": "2.29.4",
     "moment-timezone": "^0.5.37",
     "multer": "^1.4.5-lts.1",
     "node-blob": "^0.0.2",
-    "node-fetch": ">=3.2.10",
+    "node-fetch": ">=2.6.7",
     "nunjucks": "^3.2.2",
     "nunjucks-date-filter": "^0.1.1",
     "passport-azure-ad": "4.3.3",
```

Comment:
Just wondering why two of these have been down graded compare to master?

Reply:
revert the change. frontend was broken on my machine, so trying to fix it and mistakenly push my packages.json file changes.